andela/ah-kifaru-frontend
# #167998283 User can view a single article

#### What does this PR do?
- Create a single article page
- Fetch a single article from axios endpoint with slug and display on a single article page.
#### Description of Task to be completed?
- Create a single article pages and fetch api for single article using article slug 
#### How should this be manually tested?
- Clone the project then run command `npm run start:dev`
- Navigate to `localhost:3000/slug` to see a single article.
#### Any background context you want to provide?
- none
#### What are the relevant pivotal tracker stories? (if applicable)
[#167998283](https://www.pivotaltracker.com/story/show/167998283)
